### PR TITLE
fix imports

### PR DESCRIPTION
**Pull request by azazellochg**

### CODE REVIEW

File: scipion/install/inspect-plugins.py [hunk 1]
```diff
@@ -39,7 +39,7 @@
 from pyworkflow.protocol import Protocol
 import pyworkflow.utils as pwutils
 
-from plugin_funcs import PluginInfo
+from .plugin_funcs import PluginInfo
```
Comment (pconesa) [inline scipion/install/inspect-plugins.py]: Thanks @azazellochg . THere is something weird with this imports. 
  File "/home/pablo/desarrollo/scipion-app/scipion/install/install-plugin.py", line 33, in <module>
    from .plugin_funcs import PluginRepository, PluginInfo
SystemError: Parent module '' not loaded, cannot perform relative import

I guess we need to import install. But I keep removing some relative imports while other add them...Not sure is I have something worng locally or what.

Comment (pconesa) [reply scipion/install/inspect-plugins.py]: What works for me is an absolute import like this:
from scipion.install.plugin_funcs import PluginRepository, PluginInfo
from scipion.install.script import defineBinaries

Comment (azazellochg) [reply scipion/install/inspect-plugins.py]: ok, I can reproduce it, let me have a look

Comment (pconesa) [reply scipion/install/inspect-plugins.py]: may be the difference is how it is called. I always use scipion python ...../inspect_plugin.py

Comment (azazellochg) [reply scipion/install/inspect-plugins.py]: I use scipion run python .../inspect-plugins.py ...

Comment (azazellochg) [reply scipion/install/inspect-plugins.py]: @pconesa  I reverted back to abs imports. Inspect is working, but don't know about install.

**Pull request merged**